benchmarks.json

```diff
@@ -17,6 +17,18 @@
       "language": "R"
     }
   },
+  {
+    "command": "csv-write ALL --iterations=3 --all=true --drop-caches=true",
+    "flags": {
+      "language": "Python"
+    }
+  },
+  {
+    "command": "csv-write ALL --iterations=3 --all=true --drop-caches=true --language=R",
+    "flags": {
+      "language": "R"
+    }
+  },
```

Comment:
Technically, we don't need to add these here — adding them here just enables them to run on our dedicated runners. But I think these are good to be running so lets keep them here.

We might need to split up the benchmarks so we don't overload those systems (though a large chunk of the benchmark time is spent dealing with duckdb for the purposes of TPC-H which ideally will go away soon...)

Reply:
I think a test fails if I don't put them here, so we'd need to go adjust whatever test that is if so